### PR TITLE
Add spotify-template responsive

### PR DESCRIPTION
**Pull request by raw-leak**

### CODE REVIEW

File: staff/mykhaylo-gusak/spotify-template/Componenets/variables.sass [hunk 1]
```diff
@@ -0,0 +1,15 @@
+$breakpoint-tablet: 700px
```
Comment (lexbarba) [inline staff/mykhaylo-gusak/spotify-template/Componenets/variables.sass:1]: @MichaelXXI nice use of variables!

File: staff/mykhaylo-gusak/spotify-template/index.html [hunk 2]
```diff
@@ -0,0 +1,501 @@
...
+    <meta http-equiv="X-EN-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+        integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <title>Spotyfy</title>
+</head>
+
+<body>
+
+    <main class="container__mobile-first">
```
Comment (lexbarba) [inline staff/mykhaylo-gusak/spotify-template/index.html:16]: @MichaelXXI good use of BEM!

**Review by lexbarba — APPROVED** [review]

File: staff/mykhaylo-gusak/spotify-template/Componenets/variables.sass (hunk 1 above)
Comment (raw-leak) [reply staff/mykhaylo-gusak/spotify-template/Componenets/variables.sass:1]: Thx 💃

File: staff/mykhaylo-gusak/spotify-template/index.html (hunk 2 above)
Comment (raw-leak) [reply staff/mykhaylo-gusak/spotify-template/index.html:16]: Thx 💃

**Pull request merged**